docs/user-guide/using-jwt-token-with-apiml.md

```diff
@@ -0,0 +1,202 @@
+# Using JWT token with API Mediation Layer
+
+:::info**Required roles:** system administrator, security administrator
+:::
+
+## Authentication with JWT Token
+
+In Zowe, authentication can be performed via JWT tokens, whereby a token can be provided by a specialized service, which can then be used to provide authentication information. This service is described in more details at [Zowe Authentication and Authorization Service](https://github.com/zowe/api-layer/wiki/Zowe-Authentication-and-Authorization-Service). When a client authenticates with the API ML, the client receives the JWT token in exchange. This token can be used for further authentication. If z/OSMF is configured as the authentication provider and the client already received a JWT token produced by z/OSMF, it is possible to reuse this token within API ML for authentication.
+
+This parent article describes how services in the Zowe API ecosystem are expected to accept and use JWT tokens so that API clients have a stadardized experience. 
+
+The JWT tokens are by default produced by z/OSMF and the API Mediation Layer serves only as proxy. For information about how to change who and how tokens are produced, see [Authentication Providers within Enable Single Sign On for Clients](../user-guide/api-mediation/configuration-jwt/#saf-as-an-authentication-provider)
+
+TODO: https://docs.zowe.org/stable/extend/extend-apiml/authentication-for-apiml-services/#authentication-providers should move to the https://docs.zowe.org/stable/user-guide/api-mediation/configuration-jwt 
+
+### Token-based Login Flow and Request/Response Format
+
+The following sequence describes how authentication through JWT tokens works:
+
+1. The API client obtains a JWT token by using the POST method on the `/auth/login` endpoint of the API service that requires a valid user ID and password.
+
+2. The API service then calls an *authentication provider* that returns a JWT token that contains the user ID claim in the HTTP cookie named `apimlAuthenticationToken` with attributes `HttpOnly` and `Secure`.
+
+3. The API client stores the JWT token or cookie and sends the token with every request as a cookie with the name `apimlAuthenticationToken`.
+
+#### Obtaining the token
+
+<!-- We should outline how to obtain a token in a sequence of numbered steps. -->
+
+- The full URL is the base URL of the API service plus `/auth/login`. If the application has the base URL with `/api/v1`, the full URL could have the format: `https://hostname:port/api/v1/auth/login`.
```

Comment:
The full path for API ML will be /gateway/auth/login - The SDK has a bit of difference.